jekyll/__glossary/user.md

```diff
@@ -0,0 +1,5 @@
+---
+term: User
+---
+
+An individual user within an org. A CircleCI user is anyone that can log in to the CircleCI platform with a username and password. Users must be added to a GitHub or Bitbucket org to view or follow associated CircleCI projects.  Users may not view project data that is stored in environment variables.  
```

Comment:
2nd sentence: "that" --> "who"

"A CircleCI user is anyone who can log in to CircleCI with a username and password."

3rd sentence: trim

"Users may not view project data stored in environment variables."

Reply:
thanks fixed first sentence. keeping the third for clarity. thank you!